diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocComments.kt

```diff
@@ -79,13 +82,32 @@ class KdocComments(private val configRules: List<RulesConfig>) : Rule("kdoc-comm
                 FILE -> checkTopLevelDoc(node)
                 CLASS -> checkClassElements(node)
                 VALUE_PARAMETER -> checkValueParameter(node)
+                PRIMARY_CONSTRUCTOR -> checkParameterList(node.findChildByType(VALUE_PARAMETER_LIST))
                 else -> {
                     // this is a generated else block
                 }
             }
         }
     }
 
+    private fun checkParameterList(node: ASTNode?) {
+        val kdocBeforeClass = node
+            ?.parent({ it.elementType == CLASS })
+            ?.findChildByType(KDOC) ?: return
+        val propertiesInKdoc = kdocBeforeClass
+            .kDocTags()
+            ?.filter { it.knownTag == KDocKnownTag.PROPERTY}
+            ?.mapNotNull {it.getSubjectName()}
+        val propertyNames = (node.psi as KtParameterList)
+            .parameters
+            .mapNotNull { it.nameIdentifier?.text }
+        propertiesInKdoc?.let { kdocProperties ->
```

Comment:
`let` is not needed here, you can just safe-access call methods on `propertiesInKdoc`